plugins/ImageReader/ImageReader.py

```diff
@@ -124,8 +128,15 @@ def _generateSceneNode(self, file_name, xz_size, peak_height, base_height, blur_
 
             Job.yieldThread()
 
-        height_data *= scale_vector.y
-        height_data += base_height
+        if use_transparency_model:
+            p = 1.0 / math.log(transmittance_1mm / 100.0, 2) # base doesn't matter here. use base 2 for fast computation
+            min_luminance = 2.0 ** ((peak_height - base_height) / p)
+            for (y, x) in numpy.ndindex(height_data.shape):
+                mapped_luminance = min_luminance + (1.0 - min_luminance) * height_data[y, x]
+                height_data[y, x] = base_height + p * math.log(mapped_luminance, 2) # use same base as a couple lines above this
```

Comment:
Thanks for clarifying that the base should be the same. In that case, you can just remove the second parameter so the default will be used, which is a more powerful way of saying "I don't care what base you use, as long as you are consistent."

Also, please don't use single letter variables. Because code style, but also because I actually don't know what it stands for. Sorry for not spotting it before.